UnitTests/BTDropInRequestTests.m

```diff
@@ -11,6 +12,30 @@ @interface BTDropInRequestTests : XCTestCase
 @implementation BTDropInRequestTests
 
 - (void)test_copyProperties {
+    BTThreeDSecureRequest *threeDSecureRequest = [[BTThreeDSecureRequest alloc] init];
+    threeDSecureRequest.versionRequested = BTThreeDSecureVersion2;
+    threeDSecureRequest.amount = [NSDecimalNumber decimalNumberWithString:@"1.45"];
+    threeDSecureRequest.nonce = @"a-nonce";
+    threeDSecureRequest.email = @"tester@example.com";
+    threeDSecureRequest.mobilePhoneNumber = @"5151234321";
+
+    BTThreeDSecurePostalAddress *billingAddress = [[BTThreeDSecurePostalAddress alloc] init];
+    billingAddress.givenName = @"Joe";
+    billingAddress.surname = @"Guy";
+    billingAddress.phoneNumber = @"12345678";
+    billingAddress.streetAddress = @"555 Smith St.";
+    billingAddress.extendedAddress = @"#5";
+    billingAddress.locality = @"Oakland";
+    billingAddress.region = @"CA";
+    billingAddress.countryCodeAlpha2 = @"US";
+    billingAddress.postalCode = @"54321";
+    threeDSecureRequest.billingAddress = billingAddress;
+
+    BTThreeDSecureAdditionalInformation *info = [BTThreeDSecureAdditionalInformation new];
+    info.shippingAddress = billingAddress;
+    info.shippingMethodIndicator = @"GEN";
+    threeDSecureRequest.additionalInformation = info;
+
```

Comment:
To me this will read better if the 3DS related fields (`BTThreeDSecureRequest`, `BTThreeDSecurePostalAddress`, and `BTThreeDSecureAdditionalInformation `)  fields go below the regular `BTDropInRequest` fields. This would also be consistent with the order we test them in.

Reply:
We can't move the BTDropInRequest instantiation block up because the setting of the 3DS fields requires the other code in this function to have already run. We could move line 39 up alone, but that feels a little unnecessary. Is moving this whole block what you meant, or are we misunderstanding this?